.github/actions/javascript/postTestBuildComment/postTestBuildComment.ts

```diff
@@ -5,14 +5,12 @@ import CONST from '@github/libs/CONST';
 import GithubUtils from '@github/libs/GithubUtils';
 
 function getTestBuildMessage(): string {
-    const inputs = ['ANDROID', 'ANDROID_HYBRID', 'DESKTOP', 'IOS', 'IOS_HYBRID', 'WEB'] as const;
+    const inputs = ['ANDROID_HYBRID', 'DESKTOP', 'IOS_HYBRID', 'WEB'] as const;
     const names = {
-        [inputs[0]]: 'Android',
-        [inputs[1]]: 'Android Hybrid',
-        [inputs[2]]: 'Desktop',
-        [inputs[3]]: 'iOS',
-        [inputs[4]]: 'iOS Hybrid',
-        [inputs[5]]: 'Web',
+        [inputs[0]]: 'Android Hybrid',
```

Comment:
Should we also update this to just `Android`?

Reply:
Trying to decide if we should remove AdHoc or not. I think it's a bit in flux with the separate `testBuildHybrid.yml` file. @Julesssss said he is looking into whether we can combine them or not.

Reply:
We're close to merging them and just using the AdHoc workflow. The blocker is that currently we don't allow mobile-only changes to be build without a dummy App PR. 

Switching to 'Android' now is probably fine but might confuse people used to running HybridApp.

More details [here](https://github.com/Expensify/App/issues/56848#issuecomment-2755610000).

Reply:
Ok thanks for that context, I will remove `hybrid` from most aspects of `testBuild.yml`, but I won't touch `testHybridBuild.yml` as that's being worked on separate

Reply:
It would be great if you could just test out the workflows after making these changes. As for the HybridApp workflow I'll deprecate it asap.

Reply:
Yes, I will run both once I have the changes wrapped up 👍